### PR TITLE
Add optional param

### PR DESCRIPTION
**Pull request by raluvy95**
I don't know if my code works fine. Sorry, I just edited on mobile

### CODE REVIEW

File: bot/cogs/moderation/silence.py [hunk 1]
```diff
@@ -2,6 +2,7 @@
 import logging
 import textwrap
 from datetime import datetime
+from typings import Optional
```
Comment (AtieP) [inline bot/cogs/moderation/silence.py]: This module doesn't exist. You meant `typing`.

File: bot/cogs/moderation/silence.py [hunk 2]
```diff
@@ -27,7 +28,7 @@ def __init__(self) -> None:
     async def silence(
         self,
         ctx: commands.Context,
-        until: DurationConverter
+        until: Optional[DurationConverter]="15m"
```
Comment (AtieP) [inline bot/cogs/moderation/silence.py]: `DurationConverter` returns a `datetime.datetime` object, so the default parameter also must be a `datetime.datetime` object.

**Review by AtieP — CHANGES_REQUESTED** [review]

File: bot/cogs/moderation/silence.py (hunk 2 above)
Comment (AtieP) [reply bot/cogs/moderation/silence.py]: Also, there should be a space after and before the `=`. The linter will fail, and therefore the build will also fail.

Comment (raluvy95) [reply bot/cogs/moderation/silence.py]: I didn't knew about this, sorry 😕

Comment (AtieP) [reply bot/cogs/moderation/silence.py]: Don't worry. Just fix the code and I will merge the pull request. I suggest you to use `datetime.datetime.now()` and then add 15 minutes using `datetime.timedelta` inside the parameter list to make it as a default value. Something like (for example):
```py
def silence(until = datetime.datetime.now() + 15minutes):
    pass
```

Comment (raluvy95) [reply bot/cogs/moderation/silence.py]: ok I will when I'm on pc

**Comment by raluvy95** [comment]
Why it says failed if the code looks fine? How can I check the errors there? It doesn't tell me what and which is affected changed code :confounded:

**Comment by AtieP** [comment]
@raluvy95 These are the errors:
```
./bot/cogs/moderation/silence.py:31:80: E501 line too long (85 > 79 characters)
./bot/cogs/moderation/silence.py:31:80: E251 unexpected spaces around keyword / parameter equals
./bot/cogs/moderation/silence.py:31:82: E251 unexpected spaces around keyword / parameter equals
```
The first error means that the line is very long. Try to make it shorter, like, for example,
```py
calling_functions(
    like=this
)
```
.
The second and third error mean that you have unnecessary spaces there. Make it like this `datetime.now() + timedelta(minutes=15)`.

**Comment by AtieP** [comment]
> Why it says failed if the code looks fine? How can I check the errors there? It doesn't tell me what and which is affected changed code confounded

You need to click in the details of the check.

**Pull request closed without merging**